snapshottest/formatters.py

```diff
@@ -36,6 +37,8 @@ def can_format(self, value):
         return isinstance(value, self.types)
 
     def format(self, value, indent, formatter):
+        if isinstance(value, Enum):
+            value = value.value
```

Comment:
Instead of doing this, can you create a format-func instead?
This is how I would do it:

```diff
diff --git a/snapshottest/formatters.py b/snapshottest/formatters.py
index 4d7bc1c..b721565 100644
--- a/snapshottest/formatters.py
+++ b/snapshottest/formatters.py
@@ -37,8 +37,6 @@ class TypeFormatter(BaseFormatter):
         return isinstance(value, self.types)

     def format(self, value, indent, formatter):
-        if isinstance(value, Enum):
-            value = value.value
         return self.format_func(value, indent, formatter)


@@ -106,6 +104,10 @@ def format_dict(value, indent, formatter):
     return "{%s}" % (",".join(items) + formatter.lfchar + formatter.htchar * indent)


+def format_enum(value, indent, formatter):
+    return formatter(value.value)
+
+
 def format_list(value, indent, formatter):
     return "[%s]" % format_sequence(value, indent, formatter)

@@ -164,6 +166,7 @@ class GenericFormatter(BaseFormatter):
 def default_formatters():
     return [
         TypeFormatter(type(None), format_none),
+        TypeFormatter((Enum,), format_enum),
         DefaultDictFormatter(defaultdict, format_dict),
         CollectionFormatter(dict, format_dict),
         CollectionFormatter(tuple, format_tuple),
```